### ISSUE TITLE
variables link_deno, link_510k, mapping and maybe others are missing from the code. Code doesn't run.

### ISSUE THREAD

**Issue by luckrols** [ericwu09/medical-ai-evaluation#2]
In the "Download FDA data.ipynb" file, the variables "link_deno", "link_510k", "mapping" (and maybe others?) are missing from the code, so the code doesn't run. 

I searched a bit on my own and for the link_510k and link_deno I think it should be the links below (correct?), however, for the "mapping" variable I still get :
"NameError: name 'mapping' is not defined"

Would it be possible to update the notebook files so that the missing variables are included please? How should the "mapping" variable be defined?

link_510k='https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm?ID='
link_deno='https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/denovo.cfm?id='

**Comment by WuliZhuang** [comment]
> 

hello, I still have the same question about "mapping". have you solved it yet?

**Comment by luckrols** [comment]
I have not found a solution yet. I hope the author can solve this mystery?

**Comment by JuanGitJourney** [comment]
@luckrols, @WuliZhuang I guess this might be of your interest.

mapping = {'Device Classification Name': 'device_classification_name',
          '510(k) Number': 'approval_number',
           'De Novo Number': 'approval_number',
          'Device Name': 'device_name',
          'Applicant': 'applicant_name',
           'Requester': 'applicant_name',
          'Applicant Contact': 'applicant_contact',
          'Correspondent': 'correspondent_name',
          'Regulation Number': 'regulation_number',
          'Classification Product Code': 'classification_product_code',
          'Date Received': 'date_received',
          'Decision Date': 'decision_date',
           'Decision': 'decision',
           'Classification Advisory Committee': 'classification_advisory',
           'Review Advisory Committee': 'review_advisory',
           'Regulation Medical Specialty': 'regulation_medical_specialty',
           '510k Review Panel': 'review_panel',
           'statement': 'summary_link',
           'Summary': 'summary_link',
           'FDA Review': 'summary_link',
           'Reclassification Order': 'reclass_link',
           'Type': 'type',
           'Reviewed by Third Party': 'reviewed_by_third_party',
           'Combination Product': 'combination_product'
          }

**Comment by ericwu09** [comment]
Apologies for not seeing this thread earlier, please let me know if you still have questions!

**Comment by luckrols** [comment]
> Apologies for not seeing this thread earlier, please let me know if you still have questions!

Yes please. I must admit I didn't test the solution provided by JuanGitJourney, yes, I'm still interested in the solution. Thank you.
